### ISSUE TITLE
Valid protobuf rejected by OpenCPU

### ISSUE THREAD

**Issue by vlakam** [opencpu/opencpu#341]
Hello.

Here is protobuf binary that i pass to function via OpenCPU call.
[valid.zip](https://github.com/opencpu/opencpu/files/3152401/valid.zip)
It results into 
```
OpenCPU error HTTP 400
Unparsable argument: c(["BMDM, untreated, 3"])
```
However protobuf is valid as i can unserialize it by protolite.

Example is not available atm

**Comment by jeroen** [comment]
Reprex:

```sh
curl --data-binary "@valid.bin" -H "Content-Type: application/protobuf" \
  https://cloud.opencpu.org/ocpu/library/base/R/list 
```

**Comment by jeroen** [comment]
Fixed in opencpu release 2.1.2.
